src/main/java/me/crespel/karaplan/web/api/v1/PlaylistController.java

```diff
@@ -50,19 +51,31 @@
 	public Set<Playlist> getPlaylists(@PageableDefault Pageable pageable) {
 		return playlistService.findAll(pageable);
 	}
+	
+	@GetMapping("/authorized")
+	@ApiOperation("Get all authorized playlists")
+	public Set<Playlist> getAuthorizedPlaylists(@PageableDefault Pageable pageable, Principal user) {

```

Comment:
Prefer using `@ApiIgnore @AuthenticationPrincipal(expression = "user") User user` instead of `Principal user`